Expecto.Tests/Tests.fs

```diff
@@ -184,8 +184,8 @@ let timeouts =
           match testFromType Dummy.thisModuleType.Value with
           | Some (TestList (
                       Seq.Two (
-                          TestLabel("test B", TestList _),
-                          TestLabel("test A", TestList _)))) -> ()
+                          TestLabel("test B", TestList (_, Normal), Normal),
```

Comment:
Maybe we should consider only making parts of the DU possible to focus on? Perhaps even just the tests, the list and not the label?